### PR TITLE
Update openid_signing.md

### PR DESCRIPTION
**Pull request by jonvnadelberg**
This is a fix for https://github.com/sigstore/docs/issues/163

Summary
Remove duplicate and obsolete info from file.

Release Note
NONE

Documentation
Docs only fix

### CODE REVIEW

File: content/en/cosign/openid_signing.md [hunk 1]
```diff
@@ -4,71 +4,38 @@ category: "Cosign"
...
-
-That's it! No keys!
-The rest of the flags (annotations, claims, tlog, etc.) should all work the same.
+Keyless signing associates identities, rather than keys, with an artifact signature. Fulcio issues short-lived certificates binding an ephemeral key to an OpenID Connect identity. Signing events are logged in Rekor, a signature transparency log, providing an auditable record of when a signature was created.
 
-## Overview
+See the [Fulcio repository](https://github.com/sigstore/fulcio) and [Rekor repository](https://github.com/sigstore/rekor) for more information.
 
-This uses ephemeral keys and certificates, which are signed automatically by the Fulcio root CA. Signatures are stored in the Rekor transparency log, which automatically provides an attestation as to when the signature was created.
+## Keys
```
Comment (haydentherapper) [inline content/en/cosign/openid_signing.md:15]:
```suggestion
## Root of Trust
```

**Comment by haydentherapper** [comment]
@jonvnadelberg is this the same content as in #164?

**Review by haydentherapper — DISMISSED** [review]

File: content/en/cosign/openid_signing.md [hunk 2]
```diff
@@ -4,71 +4,38 @@ category: "Cosign"
...
+To log in and set up your OIDC Identity, follow these steps:
 
-In automated environments, cosign also supports directly using OIDC Identity Tokens from specific issuers. These can be supplied on the command line with the `--identity-token` flag. The `audiences` field must contain `sigstore`.
+1. Follow the link that Cosign provides to the Issuer login page.
+2. Log in to one of the supported systems that is shown:
+![Identity Sign-in](/cosign_identity_login.png)
+3. Go back to your terminal or other application. Redirection back to Cosign is seamless.
 
-Cosign also has support for detecting some of these automated environments and producing an identity token. Currently this supports Google and Github.
+Currently Sigstore supports Microsoft, Google, and GitHub. As an alternative you can also use the environment variable `SIGSTORE_ID_TOKEN` to identify yourself. Cosign also has support for detecting some of these automated environments and producing an identity token. Currently this supports Google and GitHub, in addition to other environments. See [Cosign's providers](https://github.com/sigstore/cosign/tree/main/pkg/providers) for a complete list.
```
Comment (ltagliaferri) [inline content/en/cosign/openid_signing.md:36]: "identify yourself" sounds a little not specific enough to me, should it also reference an identity token?

```suggestion
Currently, Sigstore supports Microsoft, Google, and GitHub. As an alternative, you can also use the environment variable `SIGSTORE_ID_TOKEN` to identify yourself. Cosign also has support for detecting some of these automated environments and producing an identity token. Currently this supports Google and GitHub, in addition to other environments. See [Cosign's providers](https://github.com/sigstore/cosign/tree/main/pkg/providers) for a complete list.
```

File: content/en/cosign/openid_signing.md [hunk 3]
```diff
@@ -99,30 +66,18 @@ In order to impersonate an IAM service account, your account must have the `role
...
 
-### Custom root cert
-
-You can override the public good instance root CA using the environmental variable `SIGSTORE_ROOT_FILE`, e.g.
+### Custom roots of trust
 
-```
-export SIGSTORE_ROOT_FILE="/home/jdoe/myrootCA.pem"
-```
+For information on custom roots of trust, see [Configuring Cosign with Custom Components](https://docs.sigstore.dev/cosign/custom_components/).
```
Comment (ltagliaferri) [inline content/en/cosign/openid_signing.md:83]:
```suggestion
For information on custom roots of trust, see [Configuring Cosign with Custom Components](/cosign/custom_components/).
```

**Review by ltagliaferri — CHANGES_REQUESTED** [review]
Tiny edits / questions here

File: content/en/cosign/openid_signing.md (hunk 2 above)
Comment (jonvnadelberg) [reply content/en/cosign/openid_signing.md:36]: SHould it be "Identify yourself with an identity token"?

**Review by ltagliaferri — APPROVED** [review]

**Pull request merged**